src/__test__/components/plots-and-tables/ViolinPlotIndex.test.jsx

```diff
@@ -7,15 +7,16 @@ import { createStore, applyMiddleware } from 'redux';
 import thunk from 'redux-thunk';
 import _ from 'lodash';
 import fetchMock, { enableFetchMocks } from 'jest-fetch-mock';
-
+import '@testing-library/jest-dom';
```

Comment:
I think you can remove this. Are you using any of the [custom matchers](https://github.com/testing-library/jest-dom#usage) provided by the library?